ci/browser-tests/cypress/integration/contract_mm.ci.spec.js

```diff
@@ -0,0 +1,38 @@
+context("Contracts MM", () => {
+    beforeEach(() => {
+        cy.dummyLogin("benjamin");
+        cy.selectLanguage("#EN");
+    });
+
+    it("finds contracts with search and navigates there + edits details + project link", () => {
```

Comment:
I changed the .click() ones to .click({force: true}) in the followon PR (https://github.com/solita/mnt-teet/pull/702)